cmd/admin/rollback/rollback.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*/
+Package main of cmd/admin/rollback is a command line utility that enables a named
+network change to be rolled back.
+
+If no name is given that last network change is rolled back
+ */
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"github.com/onosproject/onos-config/pkg/northbound"
+	"github.com/onosproject/onos-config/pkg/northbound/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"log"
+)
+
+func main() {
+	address := flag.String("address", ":5150", "address to which to send requests")
+	changeName := flag.String("changename", "", "")
+	keyPath := flag.String("keyPath", "", "path to client private key")
+	certPath := flag.String("certPath", "", "path to client certificate")
+	flag.Parse()
+
+	var opts = []grpc.DialOption{}
+	if *keyPath != "" && *certPath != "" {
```

Comment:
we ought to extract lines 43 to 60. using them a bit all over the place

Reply:
This is targeted by Adib with #173 and #174